app/containers/ConfigurationsPage.js

```diff
@@ -55,7 +44,7 @@ const ConfigurationsPage = ({history, configurations, searchQuery, onSearchChang
                     <ConfigElem key={config} item={config}/>
                 ))}
             </ConfigList>
-
+            <Button onPress={() => changeActiveConfig("Maciej Paciej")}><Text>Zmień aktywną konfigurację</Text></Button>
```

Comment:
No tak, póki nie ma implementacji wybierania aktywnej konfiguracji, to tak żeby sprawdzić czy działa akcja.